db/migrate/20200302173620_drop_end_user_trigger_procedure.rb

```diff
@@ -0,0 +1,6 @@
+class DropEndUserTriggerProcedure < ActiveRecord::Migration[5.0]
+  def change
+    return unless System::Database.postgres?
+    System::Database::Postgres::TriggerProcedure.new('tp_end_user_plans_tenant_id', nil).drop
```

Comment:
maybe @guicassolato or @duduribeiro know why and the best way to fix it, but for me, it only worked like this:
```ruby
require 'system/database/postgres'

class DropEndUserTriggerProcedure < ActiveRecord::Migration[5.0]
  def change
    return unless System::Database.postgres?
    System::Database::Postgres::TriggerProcedure.new('tp_end_user_plans_tenant_id', nil).drop
  end
end
```

This is why (showing it from the Rails console):
![image](https://user-images.githubusercontent.com/11318903/75742925-7d854500-5d0f-11ea-86b0-e70519773f9b.png)